samples/local_attestation/enc1/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+add_executable(enclave_a ecalls.cpp)
+target_link_libraries(enclave_a common)
+target_include_directories(enclave_a PRIVATE ${CMAKE_BINARY_DIR}/enc1)
```

Comment:
I messed things a bit here because I was confused by:
https://github.com/Microsoft/openenclave/blob/master/samples/local_attestation/Makefile#L6-L8
 and thought it was a good idea to make it top level.

I will update the to be per enclave basis.

Also should I send out a PR to update the Makefiles as well? The top level genkey target is not used.

Reply:
If it can be per enclave, it makes some things easier; but sometimes they can't be.

If you find stuff that can be fixed in the Makefiles, go for it! PRs are welcome.

Reply:
I will send out a PR to fan out those targets from the Makefiles.

I updated the this PR with the key modifications per enclave can you PTAL?